index.html

```diff
@@ -603,3 +612,5 @@
 [fire a simple event]: https://html.spec.whatwg.org/multipage/webappapis.html#fire-a-simple-event
 [feature-detect]: https://gist.github.com/igrigorik/a02f2359f3bc50ca7a9c
 [authoritative]: https://httpwg.github.io/specs/rfc7540.html#authority
+[unsupported MIME type]: https://mimesniff.spec.whatwg.org/#supported-by-the-user-agent
+[parsable MIME type]: https://mimesniff.spec.whatwg.org/#parsable-mime-type
```

Comment:
Maybe export these definitions in MIME Sniffing first? MIME Sniffing is indexed in the "global" definition index so if they're simply not exported that should be easy to fix.